src/index.html

```diff
@@ -11,8 +11,79 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,
+    wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;
+    1,500;1,700;1,900&family=Space+Grotesk:wght@300..700&family
+    =Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img">
+        <a
+          href="#"
+          class="card__img-img"
+        ></a>
+      </div>
+
+      <h1 class="card__header">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__response">
+        <div class="card__response-starsLine£ stars">
```

Comment:
The `stars` block is currently implemented within the `card` block. According to the task requirements, the `stars` block should be a separate BEM block with its own SCSS file. Please separate the `stars` block implementation.